src/Middleware/ActionCaller.php

```diff
@@ -31,8 +31,6 @@ public function __construct(string $class, string $method, ContainerInterface $c
     public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
         $controller = $this->container->get($this->class);
-        $response = (new Injector($this->container))->invoke([$controller, $this->method], [$request, $handler]);
```

Comment:
Maybe it is still necessary because we have the ability to add ActionCaller in random order.

Reply:
The line was always calling a handler but not using result of the call. If you want to delegate handling to the handler, you can do it in the action code if needed.